requirements/AUR-CBI-INT-0070/standalone-project/Makefile

```diff
@@ -0,0 +1,29 @@
+export CC = gcc
+export CFLAGS = -O0 -g -fno-omit-frame-pointer
+export LIBS =
+export LDFLAGS =
+
+export OBJDIR = "$(shell pwd)/objs"
+export LIBDIR = "$(shell pwd)/libs"
+export BINDIR = "$(shell pwd)/bin"
+
+SUBDIRS = src
+SUBDIRS_CLEAN = $(SUBDIRS%:=% clean)
+
+all: dirs $(SUBDIRS)
+
+$(SUBDIRS):
```

Comment:
This is not requested explicitly, but for the sake of outside reviewers, it would be very appreciated, if, in the requirement folder (AUR-CBI-INT-0070) there would be a short text file which describes the relevant content and process:

TASTE project was created, with 2 functions
QGen function contains dummy C implementation to allow compilation in environment without access to Simulink and QGen
Standalone project was created by extracting the 2 functions and hooking up the necessary interfaces to facilities provided by the standalone application,
The test can be executed by running this or that command.

Reply:
Done